src/app/screens/sendBtc/index.tsx

```diff
@@ -154,7 +157,7 @@ function SendBtcScreen() {
     ];
     setRecipient(recipients);
     if (validateFields(address, amountToSend)) {
-      mutate({ recipients });
+       mutate({ recipients, seedPhrase });
```

Comment:
Space at start 👀 You should probably enable format on save as per the Vscode page in Notion

Reply:
done